extensions/android-paging3/src/test/java/com/squareup/sqldelight/android/paging3/KeyedQueryPagingSourceTest.kt

```diff
@@ -0,0 +1,243 @@
+package com.squareup.sqldelight.android.paging3
+
+import androidx.paging.PagingConfig
+import androidx.paging.PagingSource.LoadParams.Refresh
+import androidx.paging.PagingSource.LoadResult
+import androidx.paging.PagingState
+import com.squareup.sqldelight.Query
+import com.squareup.sqldelight.Transacter
+import com.squareup.sqldelight.TransacterImpl
+import com.squareup.sqldelight.android.AndroidSqliteDriver
+import com.squareup.sqldelight.db.SqlCursor
+import com.squareup.sqldelight.db.SqlDriver
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+import kotlin.test.assertNull
+
+@ExperimentalCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+class KeyedQueryPagingSourceTest {
+
+  private lateinit var driver: SqlDriver
+  private lateinit var transacter: Transacter
+
+  @Before fun before() {
+    driver = AndroidSqliteDriver(
```

Comment:
if you want to use jvm instead of robolectric this should just be `JvmSqliteDriver` and then you need to manually call `create` like [this](https://github.com/cashapp/sqldelight/blob/master/sqldelight-gradle-plugin/src/test/integration/src/test/java/com/squareup/sqldelight/integration/IntegrationTests.kt#L31-L32)

Reply:
Done! thanks for the reference!